app/matplotlib.ipynb

```diff
@@ -159,20 +159,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
-   "outputs": [],
    "source": [
     "data = np.random.normal(0, 1, 1000)\n",
     "\n",
-    "# write your code here\n"
+    "plt.hist(data, bins=5, color=\"red\")\n",
```

Comment:
The bin size for the histogram is quite wide with only 5 bins. Consider increasing the number of bins to make them narrower, as per the task requirement.